cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/internal/SimpleSSA.qll

```diff
@@ -1,84 +0,0 @@
-import SimpleSSAInternal
```

Comment:
@dave-bartolomeo is this deletion correct?

Reply:
Both `SimpleSSA.qll` and `SimpleSSAInternal.qll` should have been completely deleted. I think @adityasharad did a force-push after you commented, fixing this.